src/XliffTasks.Tests/ResxDocumentTests.cs

```diff
@@ -45,7 +45,42 @@ public void BasicLoadAndTranslate()
             document.Translate(translations);
             document.Save(writer);
 
-            Assert.Equal(expectedTranslation, writer.ToString());
+            AssertHelper.AssertWithoutLineEndingDifference(expectedTranslation, writer.ToString());
         }
+
+        [Fact(DisplayName = "delete nodes that point to external files which relative path cannot be easily adjusted")]
```

Comment:
Grammar: "which relative paths cannot" --> "with relative paths that cannot"

Reply:
I've never seen DisplayName used like this before. We don't do it anywhere else. It's going to be jarring in test explorer if the others are method names. 

I think I prefer longer PascalCased method names because I can double click them in log output and then Ctrl+C to go searching for it.  ;)

Reply:
"I prefer longer PascalCased method names" me too, until there is long path exception

Reply:
These tests don't write to disk, though.